README.md

```diff
@@ -66,7 +67,7 @@ with your default Python environment and can easily use different versions of Py
 in different projects. You can create a virtual environment for ELFI using anaconda with:
 
 ```
-conda create -n elfi python=3.5 numpy
+conda create -n elfi python=3.5 matplotlib
```

Comment:
Matplotlib depends on numpy, so that will be installed as well.

The point is not to have matplotlib in Travis...?